server/src/main/java/io/spine/server/entity/Repository.java

```diff
@@ -104,6 +104,7 @@ protected Repository() {
      *
      * @param obj an instance to store
      */
+    @SuppressWarnings("AbstractMethodWithMissingImplementations") // work-around IDEA bug.
```

Comment:
Please suppress it with a `noinspection` comment then. It's strange that no static analyzer complains about the redundant suppression.